sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/policy/RequestRetryPolicy.java

```diff
@@ -110,7 +110,7 @@ Clone the original request to ensure that each try starts with the original (unm
         /*
         Update the RETRY_COUNT_CONTEXT to log retries.
          */
-        context.setData(HttpLoggingPolicy.RETRY_COUNT_CONTEXT, attempt + 1);
+        context.setData(HttpLoggingPolicy.RETRY_COUNT_CONTEXT, attempt);
```

Comment:
Was there a bug that was found having an off-by-one retry count? For example, the first retry used `attempt = 2`?

Reply:
That's correct. The first attempt was coming in as `attempt = 2`. Since this only affected some log statements and doesn't affect any sdk logic as far as I know, I didn't think it really needed to go in the changelog or anything. But I can add it if we decide that's preferable.